Add spacing for long Invoice ID's

user8446 opened the pull request:
This just simply adds a space between "Reference:" and long Invoice ID's printed on the invoice. If it is long enough it appears on the invoice as:

`Reference:MyInvoiceID`

Without the space:

![Screenshot 2023-12-22 165516](https://github.com/Gnucash/gnucash/assets/18019245/69716152-7db7-4b52-9e95-013d35fd8f94)

---

With the space:

![Screenshot 2023-12-22 165350](https://github.com/Gnucash/gnucash/assets/18019245/4d0ff4ba-7518-439c-983f-07ca7c5a9b17)

user8446 committed:
Update invoice.scm

Add spacing for long Invoice ID's (Displayed as "Reference" on the Invoice)

christopherlam commented:
This breaks translations for strings. It would be much better to improve the css.

user8446 committed:
Update invoice.scm

Add spacing for long Invoice ID's w/ CSS (spacing w/ HTML in strings breaks translations)

user8446 reviewed (commented):
Thanks, I didn't think about that. Here is the spacing via CSS

christopherlam commented:
Thanks, this works!
